lib/api/controllers/authController.js

```diff
@@ -95,7 +117,7 @@ class AuthController {
   getMyRights(request) {
     return request.context.user.getRights(this.kuzzle)
       .then(rights => Object.keys(rights).reduce((array, item) => array.concat(rights[item]), []))
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Split this 98 characters long line (which is greater than 80 authorized). [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3ALineLength)